common/content/statusline.js

```diff
@@ -130,8 +130,7 @@ const StatusLine = Module("statusline", {
         if ( typeof this.setVisibility.currVisibility == 'undefined' ) {
             this.setVisibility.currVisibility = true;
             this.setVisibility.prevVisibility = true;
-            this.setVisibility.contentSeparator
-                = highlight.get('ContentSeparator').value;
+            this.setVisibility.passiveVisibility = false;
```

Comment:
If you enter the fullscreen mode, currVisibility is changed to false. Then,
if you enter the command mode, statusline appears, but currVisibility
remains false. This says that the status line should be hide when you leave
the command line. if currVisibility changes in this case, autohide is
impossible. So, currVisibility and passiveVisibility are all needed.

2016년 2월 2일 (화) 13:48, Gary Katsevman <notifications@github.com>님이 작성:

> In common/content/statusline.js
> <https://github.com/vimperator/vimperator-labs/pull/407#discussion_r51523654>
> :
>
> > @@ -130,8 +130,7 @@ const StatusLine = Module("statusline", {
> >          if ( typeof this.setVisibility.currVisibility == 'undefined' ) {
> >              this.setVisibility.currVisibility = true;
> >              this.setVisibility.prevVisibility = true;
> > -            this.setVisibility.contentSeparator
> > -                = highlight.get('ContentSeparator').value;
> > +            this.setVisibility.passiveVisibility = false;
>
> It sounds like we should just change update the value of currVisbility. I
> don't think it would be good to add yet another flag.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vimperator/vimperator-labs/pull/407/files#r51523654>.
>